playbooks/vmss_migrate.yml

```
@@ -107,7 +107,8 @@
       ansible.builtin.pip:
         virtualenv: "{{ ansible_venv_path }}"
         virtualenv_python: python3
-        name: ansible==2.10.*
+        name: ansible
+        state: latest
```

```suggestion
        state: latest
        update_only: true
```
To fix ansible-lint issue